azure_arc_data_jumpstart/aks/ARM/artifacts/DataServicesLogonScript.ps1

```diff
@@ -181,11 +181,17 @@ Write-Host "Azure Arc data controller is ready!"
 Write-Host "`n"
 
 # If flag set, deploy SQL MI
-if ( $Env:deploySQLMI -eq $true )
+if ( $Env:deploySQLMI -eq $true -and ($null -eq $env:addsDomainName -or $env:addsDomainName.Length -le 0) )
```

Comment:
I have discussed this with Lior. Want to avoid additional parameters, just relying on the domain name, as this parameter is passed only when AD auth enabled. What do you think? We can change if needed.

Reply:
I changed this to avoid confusion.